pipeline/data_merging/getMupitStructure.py

```diff
@@ -0,0 +1,134 @@
+import requests
+import json
+import sys
+import argparse
+import csv
+import time
+
+
+def parse_args():
+    parser = argparse.ArgumentParser(description='Determine correct BRCA structure from MuPIT.')
+    parser.add_argument('-i', '--input', type=argparse.FileType('r'),
+                        help='Input variants.')
+    parser.add_argument('-o', '--output', type=argparse.FileType('w'),
+                        help='Output variants.')
+    options = parser.parse_args()
+    return options
+
+
+def isMissenseSubstitution(ref, alt):
```

Comment:
This should probably be called `isPointSubstitution` (or maybe something else) as a substitution at one position doesn't imply a missense substitution